test/DynamoCoreTests/Engine/SyncDataManagerTest.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.IO;
+using System.Linq;
+using NUnit.Framework;
+
+namespace Dynamo.Tests.Engine
+{
+    [TestFixture]
+    class SyncDataManagerTest : DynamoModelTestBase
+    {
+        /// <summary>
+        /// The next test method will execute the  public SyncDataManager Clone() methof from the SyncDataManager class
+        /// </summary>
+        [Test]
+        [Category("UnitTests")]
+        public void SyncDataManagerCloneTest()
+        {
+            //Arrange
+            var openPath = Path.Combine(TestDirectory, @"core\nodeLocationTest.dyn");
+            RunModel(openPath);
+            var nodesList = CurrentDynamoModel.CurrentWorkspace.Nodes;
+
+            //Act
+            //Internally the PreviewGraphSyncData calls the SyncDataManager.Clone() method
+            CurrentDynamoModel.EngineController.PreviewGraphSyncData(nodesList, true);
+
+            //Assert
+            //Verify that at least we have 1 node
+            Assert.Greater(nodesList.Count(), 0);
```

Comment:
It doesn't look like `nodesList` is being modified inside `PreviewGraphSyncData`, so why test it here? It looks like you should test something that has changed as expected after the call to `PreviewGraphSyncData`.

Reply:
@aparajit-pratap 
After the ideas that we analyzed by Slack about how to test the Clone() method I will consider this test method for the next sprint pull request, then I will proceed to remove this test case from this pull request.

Reply:
@aparajit-pratap 
This test method (the whole file) was removed from the pull request in the commit : https://github.com/DynamoDS/Dynamo/pull/10705/commits/4d07fccfc603beb0ef3d0140a6307d3bad04df9d

Please review
cc: @QilongTang